### PR TITLE
Fixed suspendWhenHidden problem with tabset.

### PR DESCRIPTION
**Pull request by maju116**

### CODE REVIEW

File: examples/simple_tabset/app.R [hunk 1]
```diff
@@ -0,0 +1,23 @@
+library(shiny)
+library(shiny.semantic)
+library(highlighter)
+library(magrittr)
+library(plotly)
```
Comment (filipstachura) [inline examples/simple_tabset/app.R]: Do we need all these packages? `highlighter`?   `magrittr `?

File: R/dsl.R [hunk 2]
```diff
@@ -216,20 +216,31 @@ tabset <- function(tabs,
...
     }),
     shiny::tags$script(paste0(
-      "$('#", id, ".menu .item').tab({onVisible:",
-      " function() {$(window).resize()} });"))
+      "/* Code below is needed to trigger visibility on reactive Shiny outputs. */
+       /* Thanks to that users do not have to set suspendWhenHidden to FALSE.   */
+       var previous_tab;
+       $('#", id, ".menu .item').tab({
+       onVisible: function(target) {
+        if (previous_tab) {
```
Comment (filipstachura) [inline R/dsl.R]: Please fix JS indentation (2 spaces).

Comment (maju116) [reply R/dsl.R]: done

File: examples/simple_tabset/app.R (hunk 1 above)
Comment (maju116) [reply examples/simple_tabset/app.R]: done

**Pull request merged**